src/remote_gym/remote_environment_management.py

```diff
@@ -30,7 +40,7 @@ def start_as_remote_environment(
     NOTE: Use the RemoteEnvironment class to connect to a remotely started environment and provide a gym.Env interface.
 
     Args:
-        local_environment: Environment which should be ran remotely on a server.
+        default_args: The arguments passed to the environment's entrypoint/constructor.
```

Comment:
Also it is important to explain that `entrypoint` needs a method called "create_environment".